src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42IslRttSpec.groovy

```diff
@@ -317,22 +313,15 @@ class Server42IslRttSpec extends HealthCheckSpecification {
         then: "SERVER_42_ISL_RTT_OUTPUT_COOKIE and SERVER_42_ISL_RTT_INPUT rules updated according to the changes"
         and: "SERVER_42_ISL_RTT_TURNING_COOKIE is not changed"
         wait(RULES_INSTALLATION_TIME) {
-            def rules = northbound.getSwitchRules(sw.dpId).flowEntries.findAll {
-                (it.cookie in [Cookie.SERVER_42_ISL_RTT_TURNING_COOKIE, Cookie.SERVER_42_ISL_RTT_OUTPUT_COOKIE]) ||
-                        (new Cookie(it.cookie).getType() in [CookieType.SERVER_42_ISL_RTT_INPUT])
-            }
+            def rules = switchRulesFactory.get(sw.dpId).getServer42ISLRules()
             assert rules.size() == northbound.getLinks(sw.dpId, null, null, null).size() + 2
             assert rules.findAll {
                 new Cookie(it.cookie).getType() == CookieType.SERVER_42_ISL_RTT_INPUT
             }*.match.inPort.unique() == [newS42Port.toString()]
             assert rules.find {
                 it.cookie == Cookie.SERVER_42_ISL_RTT_OUTPUT_COOKIE
             }.instructions.applyActions.flowOutput == newS42Port.toString()
-            assert expect(rules.find { it.cookie == Cookie.SERVER_42_ISL_RTT_TURNING_COOKIE }, sameBeanAs(s42IslRttTurningRule)
-                    .ignoring("byteCount")
-                    .ignoring("packetCount")
-                    .ignoring("durationSeconds")
-                    .ignoring("durationNanoSeconds"))
+            assert rules.find { it.cookie == Cookie.SERVER_42_ISL_RTT_TURNING_COOKIE } == s42IslRttTurningRule
```

Comment:
The same strategy was used for comparison FlowRuleEntity (EqualsAndHashCode(excludes = 'durationSeconds, durationNanoSeconds, packetCount, byteCount')
 Could not use FlowEntry as  the following fields are excluded:
  "durationSeconds", "durationNanoSeconds", "packetCount", "byteCount", "idleTimeout", "hardTimeout", "version"